Core/src/org/sleuthkit/autopsy/filesearch/DateSearchFilter.java

```diff
@@ -207,41 +204,26 @@ public void addActionListener(ActionListener l) {
 
     @Override
     @Messages ({
-        "End_Date_Before_Start_Date=The end date should be after the start date."
+        "End.date.before.start.date=The end date should be after the start date.",
+        "No.checkbox.selected=Non of the date search checkbox is selected."
```

Comment:
For the messages, we most commonly name them in the form <class name>.<optional subtype>.<name> As in Java class names, dots should only separate the main sections, not the words. So for example, "End.date.before.start" would be better as "DateSearchFilter.endDateBeforeStart"

Reply:
@zhhl and @APriestman, Ann is correct. I would go one step further and do names like DateSearchFilter.errMsg.endDateBeforeStart or DateSearchFilter.errorMessage.endDateBeforeStart